internal/slicer/slicer.go

```diff
@@ -280,10 +303,21 @@ func Run(options *RunOptions) (*Report, error) {
 		}
 		return err
 	}
+	createFile := func(opts *fsutil.CreateOptions) error {
+		if opts.Mode.IsDir() {
+			return fmt.Errorf("internal error: cannot create or modify a directory from a Starlark script")
```

Comment:
I believe we refer to those as simply "scripts" instead of "Starlark scripts", but there's another issue here: this is a createFile call, so the error is not that this is trying to modify a directory from a script, but rather tha this is a directory at all.  In either case, per earlier comment we need to talk about this interface as it's not clear at this point that we want to have ContentValue becoming a long set of callbacks for every possible way it touches the filesystem. We already have other interfaces for this.

Reply:
This error does not exist anymore now that we have reengineered the "interface".